modules/ROOT/pages/kubernetes/operations/backup-restore.adoc

```diff
@@ -33,8 +33,38 @@ To perform a backup of a Neo4j database to any cloud provider (AWS, GCP, and Azu
 ----
 helm repo update
 ----
+. Configure the credentials to access the cloud provider bucket by either using a service account or a Kubernetes secret:
++
+Service account::
+In some deployment situations, it may be desirable to assign a Kubernetes Service Account to the Neo4j pod.
+For example, if processes in the pod want to connect to services that require Service Account authorization.
+To configure the Neo4j pod to use a Kubernetes service account, set `podSpec.serviceAccountName` to the name of the service account to use.
```

Comment:
set 'serviceAccountName' , no podSpec in neo4j-admin helm chart

Reply:
good catch